app/src/main/java/com/example/nsc_events/MainActivity.kt

```diff
@@ -23,7 +39,59 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    ScreenMain()
+                    //ScreenMain()
```

Comment:
@theGaryLarson I see here that ScreenMain() is bypassed to show the scaffold for navigation but my concern is that the resulting merge will cause main to lose most of its functionality unless the person who pulls uncomments.